experimental/Schemes/AffineSchemes.jl

```diff
@@ -237,7 +237,7 @@ function hypersurface_complement(X::Spec{BRT, BRET, RT, RET, MST}, f::RET; keep_
         if issubset(Jsat, saturated_ideal(IX))
           for o in keys(DIX)
             gb = DIX[o]
-            groebner_bases(J)[o] = LocalizedBiPolyArray(localized_ring(W), singular_gens(gb), shift(gb), true)
+            groebner_bases(J)[o] = LocalizedBiPolyArray(localized_ring(W), singular_gens(gb), shift(gb), o.o, true)
```

Comment:
I'm not sure, but didn't the constructor for `LocalizedBiPolyArray` which digests a `Singular`-ideal automatically read the ordering from the Singular-side? Or did you also change that constructor?

Reply:
Yes, I had to change the constructor. I couldn't find a method that converts a Singular ordering into an  Oscar one. Before `Singular.ordering_as_symbol` was used which does not work for weighted orderings.

Reply:
Alright, that makes sense.